op-guide/tidb-v2.0-upgrade-guide.md

```diff
@@ -8,6 +8,10 @@ category: deployment
 
 This document describes how to upgrade from TiDB 1.0 or TiDB 2.0 RC version to TiDB 2.0 GA version.
 
+## Precaution
+
+Do not perform any DDL requests during the upgrading process, or else the undefined behavior error might occur.
```

Comment:
For similar conditions, we usually use the word "otherwise" instead of "or else". It'd be better to keep them consistent:)